src/ui/uploader.js

```diff
@@ -28,13 +28,20 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         members: {
             currentFile: null
         },
-        // Holds a fileObjectURL for file preview purposes
         model: {
-            fileObjectURL: null,
-            fileDetails: null
+            // uploadState can be one of the following values:
+            // "ready" (the initial state), "uploading", "errorReceived"
+            uploadState: "ready",
+            fileObjectUrl: null, // for the file preview
+            previousFileObjectUrl: null, // for file deletion on change/upload
+            serverUploadUrl: "/stories/",
```

Comment:
Some thoughts from the discussion with @BlueSlug and @jobara:

1. The upload url and method should be configurable;
2. These values make more sense to be in the "members" option rather than "model".